subprojects/docs/src/samples/userguide/tasks/addToTaskContainer/addToTaskContainer.sample.conf

```diff
@@ -0,0 +1,3 @@
+executable: gradle
+args: tasks
+expect-failure: false
```

Comment:
seems like there's a lot of duplication in these .conf files.

Reply:
I have removed all redundant lines.